src/application.coffee

```diff
@@ -220,13 +218,12 @@ fetch = (app, { deltaSource, setDeviceUpdateState = true } = {}) ->
 				if conf['RESIN_SUPERVISOR_DELTA'] == '1'
 					logSystemEvent(logTypes.downloadAppDelta, app)
 					deltaOpts = {
-						uuid, apiKey
+						uuid, apiKey, deltaSource
 						# use user-defined timeouts, but fallback to defaults if none is provided.
 						requestTimeout: checkInt(conf['RESIN_SUPERVISOR_DELTA_REQUEST_TIMEOUT'], positive: true)
-						applyTimeout: checkInt(conf['RESIN_SUPERVISOR_DELTA_APPLY_TIMEOUT'], positive: true) ? DEFAULT_DELTA_APPLY_TIMEOUT
```

Comment:
What happens now if `RESIN_SUPERVISOR_DELTA_APPLY_TIMEOUT` isn't provided, or isn't a number?

Reply:
It’ll be undefined which will result in docker-delta’s default timeout be applied, which is 0, meaning “no timeout”. That’s the behaviour we want by default, given that any timeout value is just arbitrary. If however this causes issues in specific cases, the env var can be used to adjust it.

Reply:
Ok